packages/ui/jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
   ],
   moduleNameMapper: {
-    '\\.(css|scss)$': '<rootDir>/__mocks__/styleMock.js',
+    '\\.(css|scss)$': 'identity-obj-proxy',
```

Comment:
Why you used that, is it worth adding an additional package?

Reply:
@tiltom yes, it's worth it, for instance, we can tell Jest to mock this object as imported and then all our classNames lookups on the imported styles object will be returned as is. so I can test all classNames in the component.